contribute/en/Style-and-Tone.md

```diff
@@ -89,12 +89,16 @@ Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open
 
 “Person” A human connected to a server (may be on multiple channels). “People on the server”, “People who have muted themselves” (not “Musicians” or "Channels").
```

Comment:
```suggestion
“Person” A human connected to a server (may be on multiple channels). We might say _people on the server_, or _people who have muted themselves_, rather than _Musicians_ or _Channels_.
```